pkg/sources/udsource/utils/offset.go

```diff
@@ -56,12 +60,9 @@ func (s *simpleSourceOffset) NoAck() error {
 }
 
 func ConvertToSourceOffset(offset isb.Offset) *sourcepb.Offset {
+	decoded, _ := base64.StdEncoding.DecodeString(offset.String())
```

Comment:
Could DecodeString ever fail?

Reply:
With us controlling the encoding, `DecodeString` should never fail.